From 1cf24133f9f0efe1d8a770b6200b5d5ce5fe165e Mon Sep 17 00:00:00 2001
From: Jukka Rissanen <jukka.rissanen@linux.intel.com>
Date: Wed, 23 Jul 2014 10:43:09 +0300
Subject: [PATCH 2/6] kernel: Path to config fragments was missing

This allows the layer to be used with other layers (like meta-eca).
---
 meta-iot-devkit/recipes-kernel/linux/linux-yocto-quark_3.8.bbappend | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-iot-devkit/recipes-kernel/linux/linux-yocto-quark_3.8.bbappend b/meta-iot-devkit/recipes-kernel/linux/linux-yocto-quark_3.8.bbappend
index 685730a..fa34414 100644
--- a/meta-iot-devkit/recipes-kernel/linux/linux-yocto-quark_3.8.bbappend
+++ b/meta-iot-devkit/recipes-kernel/linux/linux-yocto-quark_3.8.bbappend
@@ -10,7 +10,7 @@ module_autoload_pppox = "pppox"
 module_autoload_pppoe = "pppoe"
 
 # find defconfig path
-FILESEXTRAPATHS := "${THISDIR}/${PN}"
+FILESEXTRAPATHS := "${THISDIR}/${PN}/:${THISDIR}/${PN}/iot-devkit/"
 
 SRC_URI += "file://devkitcamera.cfg"
 SRC_URI += "file://enable_systemd.cfg"
-- 
1.8.3.1

